bin/pre-commit

```diff
@@ -11,4 +11,4 @@ fi
 
 find . \( -name '*.php' \) -exec php -lf {} \;
 
-phpunit --filter WordPress "$phpcs_root/tests/AllTests.php"
+PHPCS_DIR=$phpcs_root phpunit --bootstrap="./Test/phpcs3-bootstrap.php" --filter WordPress "$phpcs_root/tests/AllTests.php"
```

Comment:
Just making sure: did the updated instructions in the `Contributing.md  document help clear up the confusion ? If not, what can be improved both in this message and in the contributing document ?